parentChild/Parentcard.tsx

```diff
@@ -0,0 +1,58 @@
+import {  Grid, Typography,Paper, IconButton, InputAdornment, OutlinedInput, Checkbox, FormControlLabel, Button, Link, Divider, TextField } from '@mui/material'
+import React from 'react'
+import './styles.css'
+import {  SpaceBar, ViewColumn, Visibility, VisibilityOff } from '@mui/icons-material'
+import GoogleIcon from '@mui/icons-material/Google';
+import GitHubIcon from '@mui/icons-material/GitHub';
+export default function Parentcard() {
+    const paperstyle={
+        padding: '20px 30px',
+        width:300,
+        margin:"20px auto"
+    }
+    const [showPassword, setShowPassword] = React.useState(false);
+    const handleClickShowPassword = () => setShowPassword((show) => !show);
+  const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+    event.preventDefault();
+  };
+  return (
+    <Grid className='card'  >
+    <Paper className='paperstyle' style={paperstyle}>
+        <Grid container  direction='column' padding={1}>
+        <Typography variant='h5' className='head' sx={{text:'bold'}}>Sign in</Typography>
+            <Typography variant='caption'>Please enter your login credentials</Typography>
```

Comment:
Messages should not be harcoded in jsx
Try to create a constants.ts file and export messages from there